src/App.vue

```diff
@@ -1,17 +1,194 @@
 <template>
   <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+    <section id="game_canvas">
+      <Header v-bind:startGame="startGame" 
+              v-bind:isRestartButtonDisabled="isRestartButtonDisabled"/>
+      <RhymeCard v-bind:stylingObject="stylingObject"
+                 v-bind:isGameStarted="isGameStarted"
+                 v-bind:assignListeners="assignListeners"
+                 v-bind:RhymusGame="RhymusGame"
+                 v-bind:puzzlesArray="puzzlesArray"
+                 v-bind:cardNumber="cardNumber"
+                 v-bind:classNameObject="classNameObject"
+                 v-bind:guessValue="guessValue"
+                 v-bind:hintText="hintText"
+                 v-bind:wrongText="wrongText"
+                 v-bind:timeRemaining="timeRemaining"
+                 v-on:input="guessValueUpdate"
+                 v-bind:totalWrong="totalWrong"
+                 v-bind:totalCorrect="totalCorrect"
+                 v-bind:isFirstGame="isFirstGame" />
```

Comment:
We can see here that not utilizing Vuex gives us a lot of binding-heavy components. The `v-bind`s and `v-on`s can be shortened to `:` and `@` respectively, but I think it's good to leave them as is for any other who come along and read the code.